### PR TITLE
Clown college part II

### PR DESCRIPTION
**Pull request by flrsh**
## About the PR

Clowns don't strive for perfection. Perfection is useless. They just strive for growth and improvement. This is what this PR is about (no I didn't just forget to fix some stuff). 

## Why's this needed?

Honk.

### CODE REVIEW

File: code/datums/jobs.dm [hunk 1]
```diff
@@ -849,17 +850,23 @@
 		src.access = get_access("Staff Assistant")
 		return
 
+	/datum/job/civilian/staff_assistant/clown_college_student
```
Comment (pali6) [inline code/datums/jobs.dm:853]: This seems like bad indentation.

Comment (flrsh) [reply code/datums/jobs.dm:853]: Whoops!

**Comment by arch-linux** [comment]
image

Still cannot move the Job to low priority. Or is that what you were saying in 5cd78a4.
"Changelog, fix low priority job bug (just because the limit is < 0 doesn't mean it should be shoved in there". 

If that was the intent, then please ignore this.

**Comment by flrsh** [comment]
Oh, for ` fix low priority job bug`, I was referring to a bug (well, it was intentional, but I considered it a bug and so removed it) where jobs with a limit < 0 (such as -1) would be added to the low priority job list, which the game would then choose from to spawn people as. This allowed people to be clowns without having their clown college diploma (as the limit for clowns was set to -1). When you say you still cannot move the job to low priority, which job are you referring to? Staff assistant?

**Comment by arch-linux** [comment]
I am referring to the Clown College Student, as it appears in the job selection; it physically is missing the button to move it to low priority, as if it were like the staff assistant; which cannot be moved to low priority. See picture above.

**Comment by flrsh** [comment]
Oh, yeah, it's intended that it can't be moved to unwanted. Clown college student is a child of staff assistant.

**Pull request merged**